pleskdistup/common/src/writers.py

```diff
@@ -18,9 +18,15 @@ def __exit__(self, *args):
 
 
 class StdoutWriter(Writer):
+    def __init__(self):
+        self.out = open(1, 'w', closefd=False)
```

Comment:
Are we actually sending non-ASCII characters to StdoutWriter?

Reply:
Yes, for example, this can happen when we describe the pre-checker results, also known as "required conditions."

Reply:
Could you provide an example?

Reply:
Sure. This is the STR from the bug we are fixing here:
1. Add utf-8 symbol in the /etc/mail/spamassassin/init.pre. For example 'ü'
2. set locale with environment variables:
```
> LC_ALL=C
> LANG=C
```
3. Start centos2alma

Currently, the bug causes a failure while reading the file on centos2alma 1.3.0. Even if we fix the default encoding by using setlocale, it will still fail when writing the spamassassin plugin name to stdout for the same reason.